epub33/core/index.html

```diff
@@ -5301,11 +5332,22 @@ <h4>File and Directory Structure</h4>
 				</section>
 
 				<section id="sec-container-iri">
-					<h4>Relative URLs for Referencing Other Components</h4>
+					<h4>URLs in the OCF Abstract Container</h4>
 
-					<p>Files within the <a>OCF Abstract Container</a> MUST reference each other via <a
-							href="https://url.spec.whatwg.org/#relative-url-with-fragment-string"
-							>relative-URL-with-fragment strings</a> [[URL]].</p>
+					<p>The <a>container root URL</a> is the <a data-cite="url#concept-url">URL</a> [[URL]] of the
+						<a>Root Directory</a>. It is implementation specific, but MUST abide to the following requirements:</p>
+
+					<ul>
+						<li>The result of <a data-cite="url#concept-url-parser">parsing </a> "<code>/</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url">base</a> MUST be the <a>container root URL</a>.</li>
```

Comment:
Editorial: can _base_ be italicised to make it more obvious it refers to a parameter defined in the URL Parser algorithm?

This applies to various other places where the URL Parser is called.

Reply:
Yep. Done, will appear in the next commit.